apps/api/e2e/tests/agent.test.ts

```diff
@@ -953,7 +956,52 @@ describe("Agent API", () => {
     if (agentData.owner.name !== null) {
       expect(typeof agentData.owner.name).toBe("string");
     }
+  });
+
+  test.only("should not expose api key or email in public agent info", async () => {
+    // Setup admin client and login
+    const adminClient = createTestClient();
+    const adminLoginSuccess = await adminClient.loginAsAdmin(adminApiKey);
+    expect(adminLoginSuccess).toBe(true);
+
+    // Register a user and agent
+    const userName = `Get Agent Test User ${Date.now()}`;
+    const userEmail = `get-agent-${Date.now()}@example.com`;
+    const agentName = `Get Agent Test ${Date.now()}`;
+    const agentEmail = `get-agent-${Date.now()}@example.com`;
+    const agentDescription = "Agent for GET endpoint test";
+
+    const res = await registerUserAndAgentAndGetClient({
+      adminApiKey,
+      userName,
+      userEmail,
+      agentName,
+      agentDescription,
+    });
+    const agentId = res.agent.id;
+
+    // Manually update database entry to include email
+    const db = await dbManager.connect();
```

Comment:
note: iirc we want to remove `agents.email`, so i figured i'd just directly alter the db entry here vs updating all of our test helper methods